db/seeds/api.comments.model.js

```diff
@@ -16,3 +17,47 @@ exports.selectComments = (article_id) => {
     return rows;
   });
 };
+
+exports.checkArticleExists = (article_id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({ status: 404, msg: "Article not found" });
+      }
+    });
+};
+
+exports.checkUserExists = (username) => {
```

Comment:
This isn't required, if you check the seed there is a foreign key constraint on the user AND the article. If you try and post things that don't exist your psql error handler should catch these.